modules/ROOT/pages/_partials/data-sync/js-client-conflict.adoc

```diff
@@ -1,84 +1,184 @@
 
-let deleteTaskConflictResolver = (serverData, clientData) => {
-    ...
-    return serverData;
+[source,javascript]
+----
+type Mutation {
+  updateUser(id: ID!, version: String!): User
 }
+----
 
-//define a default where the clientData is used
-let defaultConflictResolver = (serverData, clientData) => {
-    return clientData
+Alternatively developers can create input elements that can be reused in every mutation and support conflict resolution.
+
+[source,javascript]
+----
+type Mutation {
+  updateUser(user: UserInput): User
+}
+----
+
+[#conflict-resolution-strategies]
+== Conflict Resolution Strategies
+
+{sync-service} allows developers to define custom conflict resolution strategies. You can provide custom conflict resolution strategies to the client in the config by using the provided `ConflictResolutionStrategies` type. By default developers do not need to pass any strategy as `UseClient` is the default. Custom strategies can also be used to provide different resolution strategies for certain operations:
+
+[source,javascript]
+----
+let customStrategy = {
+  resolve = (base, server, client, operationName) => {
+    let resolvedData;
+    switch (operationName) {
+      case "updateUser":
+        delete client.socialKey
+        resolvedData = Object.assign(base, server, client)
+        break
+      case "updateRole":
+        client.role = "none"
+        resolvedData = Object.assign(base, server, client)
+        break
+      default:
+        resolvedData = Object.assign(base, server, client)
+    }
+    return resolvedData
+  }
 }
+----
+
+This custom strategy provides two distinct strategies to be used when a conflict occurs. They are based on the name of the operation to give developers granular control. To use this custom strategy pass it as an argument to conflictStrategy in your config object:
 
-//pass it to the config object
+[source,javascript]
+----
 let config = {
 ...
-  conflictStrategy: {
-    strategies: {
-      "TaskUpdated": updateTaskConflictResolver,
-      "TaskDeleted": deleteTaskConflictResolver
-    },
-    default: defaultConflictResolver
-  }
+  conflictStrategy: customStrategy
 ...
 }
 ----
 
-NOTE: Client strategy is ignored when conflicts are resolved on the server.
-
 == Listening to Conflicts
 
-Developers can supply their own `conflictListener` implementation to get notifications about conflicts:
+{sync-service} allows developers to receive information about the data conflict that occurred between the client and the server.
+
+When a conflict occurs {sync-service} will attempt to do a field level resolution of data - meaning it will check all fields of your type to see if both the client or server has changed the same field. The client can be notified in one of two scenarios.
```

Comment:
```suggestion
When a conflict occurs, {sync-service} will attempt to do a field level resolution of data - it will check all fields of its type to see if both the client or server has changed the same field. The client can be notified in one of two scenarios.
```